recipe/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ex
+
+cd ${SRC_DIR}
+
+if [ "$(uname -m)" = "ppc64le" ]; then
+  export PYTHONUNBUFFERED=1
+fi
+
+mkdir builddir
+$PYTHON -m build --wheel --no-isolation --skip-dependency-check \
+    -Cbuilddir=builddir \
+    || (cat builddir/meson-logs/meson-log.txt && exit 1)
+$PYTHON -m pip install dist/pandas*.whl
```

Comment:
Could you point me, please, where this code come from? I didn't find it on github or docs, even conda-forge still uses `pip install .`

Reply:
When using pip install, meson-log.txt ends up in a temporary directory.
By using python-build, we can display the content of meson-log.txt on error.
scipy and numpy feedstocks have similar code in their build script.

Reply:
We can pass build options to pip too. But it's fine if we python-build.